main.js

```diff
@@ -60,4 +111,4 @@ class ModuleInstance extends InstanceBase {
 	}
```

Comment:
UpdateFeedbacks and UpdateVariableDefinitions now is undefined.

Reply:
We have no plan to support feedbacks or variables for the initial version, is it valid to initialize feedbacks and variables with empty structures ?

Reply:
You don't need to call `updateFeedbacks` or `updateVariableDefinitions` at all and in deed you are not calling these functions. But what you do is to declare them and within these function definitions you are calling some formerly importet functions from other modules/files which are now not defined any more.
Just remove the unused function definitions.
But I can tell you that feedback and presets are greatly improving user experience and so far your module is very basic.

Reply:
Thanks for the heads-up. variables and feedback modules are now temporarily disabled until  API support for feedbacks and variables are improved.